Bottom Sheet Fixes

smohamedjavid opened the pull request:
fixes #17588

### Summary

This PR fixes the following issues in the bottom sheet:

1. the sheet is cut off at the bottom in the `shell` (dark blur) theme (the drawers in the onboarding/login flow)

| Bug  | Fix
| --- | --- |
| <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/3a42fe46-4b1c-45b4-9c3e-0c963d896edb" /> | <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/9d8454c3-0807-48ae-87b5-dcdc06219edb" /> |

2. the incorrect background in the `shell` (dark blur) theme (the drawers in the onboarding/login flow)

| Bug  | Fix
| --- | --- |
| <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/8d0c2f7e-cdb7-4ad7-b237-b0cbbca72c07" /> | <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/46e4addc-6af5-4907-a160-512fe887f42c" /> |


3. the spacing at the bottom is doubled

| Bug  | Fix
| --- | --- |
| <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/d269fbc3-0b46-47d8-881d-7c64538b58b2" /> | <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/8469a1f7-3465-4ffb-97a8-43365dc4979c" /> |

4. the gradient cover is not shown in the bottom sheet

| Bug  | Fix
| --- | --- |
| <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/987c8fef-f402-465e-9c0a-21791a748f8e" /> | <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/45b767d0-5279-46a4-a7df-028b908de14f" /> |

### Review notes

The recent bottom sheet changes (d5a71e2ce08579919ee3cf8d5f9b535a5d17b91b) have broken the background of the `shell` dark blur variant and gradient cover usage. This PR fixes those bugs.

### Testing notes

The whole application needs to be tested as it updates the bottom sheet core file. A design review is not required as these are bug fixes.

### Platforms

- Android
- iOS

### Steps to test

It is advisable to test the whole application due to core file changes.

#### Gradient Cover component (not used in any screen)

- Open Status
- Navigate to `Quo2 preview > gradient > gradient cover`
- Tap on the `See in bottom sheet` button

status: ready

smohamedjavid committed:
[Fix] Bottom Sheet bottom spacing and background color

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

smohamedjavid committed:
[Fix] Gradient Cover in bottom sheet and opacity

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 43c73d84 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-17609/1/) | 2023-10-10 16:34:32 | ~6 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-231010-162819-43c73d-pr17609-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-231010-162819-43c73d-pr17609-universal.apk)|

OmarBasem commented on src/status_im2/common/bottom_sheet/view.cljs:
```diff
@@ -74,9 +74,6 @@
             bottom                  (if sheet-bottom-margin
```
I think @jo-mut was working on fixing the extra space at the bottom?

jo-mut replied:
this pr. addresses the issue https://github.com/status-im/status-mobile/pull/17547

smohamedjavid replied:
Thanks for the update @jo-mut.

I checked your PR. It resolves the bottom spacing, but the background color on the shell (dark blur) theme and gradient cover is fixed in this PR. 

I guess, I should wait for your PR to get merged. To include your fixes in this PR.

jo-mut replied:
thats fine with me thank @smohamedjavid

jo-mut replied:
@smohamedjavid https://github.com/status-im/status-mobile/blob/43c73d849c64d3c2b083a26c8e395adad71dd8bd/src/status_im2/common/bottom_sheet/view.cljs#L73

removing this code `(:bottom insets)` from the following line will solve the issue on the extra margin

jo-mut replied:
and also remove `(:bottom insets)`  in this line too and that should fix the issue in this pr
https://github.com/status-im/status-mobile/blob/43c73d849c64d3c2b083a26c8e395adad71dd8bd/src/status_im2/common/bottom_sheet/view.cljs#L74

> (:bottom insets)

smohamedjavid replied:
Thanks @jo-mut - I will add the fix in this PR 👍

status-im-auto commented:
## 77% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 10
Passed tests: 33
```
```
IDs of failed tests: 702732,703133,702957,703495,702783,703202,702786,702731,702808,702958 
```
<h3>Failed tests (10)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=17609&TR_CASE_IDS=702732,703133,702957,703495,702783,703202,702786,702731,702808,702958">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253796">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253797">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253835">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 2: Looking for a message by text: Chat is unmuted now </div></blockquote></p>

```
critical/chats/test_group_chat.py:473: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 New messages counter near chats tab button is 1 after unmute, but should be 2
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e686b1ad21d141c68d8c3dcdab1b5ca1?auth=5b1c77575b8c418f46c6d0fa6e479ebb#274">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5e4a58fa01943d99574f80f054382aa?auth=982415fd20a0ad52e2facd892fc9440c#208">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8e3d9a433454cca8eaca66bc9b9b366?auth=23a2012cb941c4066dbde2ec7cfde903#193">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253831">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 1: Wait for text element `EmojisNumber` to be equal to `1` </div><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-1']/android.widget.TextView` </div></blockquote></p>

```
critical/chats/test_group_chat.py:119: in test_group_chat_reactions
    message_element.emojis_below_message(emoji="love").wait_for_element_text(1)
../views/base_element.py:172: in wait_for_element_text
    element_text = self.find_element().text.strip()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: EmojisNumber by xpath: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-1']/android.widget.TextView` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e686b1ad21d141c68d8c3dcdab1b5ca1?auth=5b1c77575b8c418f46c6d0fa6e479ebb#149">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5e4a58fa01943d99574f80f054382aa?auth=982415fd20a0ad52e2facd892fc9440c#100">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8e3d9a433454cca8eaca66bc9b9b366?auth=23a2012cb941c4066dbde2ec7cfde903#80">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253806">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><p><blockquote><div>Device 3: Looking for a message by text: message from old member </div><div>Device 3: Looking for a message by text: message from new member </div></blockquote></p>

```
critical/chats/test_group_chat.py:309: in test_group_chat_offline_pn
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Messages PN was not fetched from offline 
```

<b>[[Data delivery issue]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e686b1ad21d141c68d8c3dcdab1b5ca1?auth=5b1c77575b8c418f46c6d0fa6e479ebb#222">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5e4a58fa01943d99574f80f054382aa?auth=982415fd20a0ad52e2facd892fc9440c#183">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8e3d9a433454cca8eaca66bc9b9b366?auth=23a2012cb941c4066dbde2ec7cfde903#156">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253802">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'test message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Delivered` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:601: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message was not delivered after resending from offline 
```

<b>[[Data delivery issue]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/02a8ca533ec944de9b41a6358e00a965?auth=00b7cb20abd86f0aac42010f735a5216#79">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c48d30fc68c242a6bdfc137fe7099eaa?auth=207a296a1e83a8474ab07a1fafd688ba#79">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253826">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `xpath`: `//*[@content-desc='password-input']/../following-sibling::*//*[@text='Join Community']` </div><div>Device 2: Tap on found: Button </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:323: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:424: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:139: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/285a1bc1174a4389b603b7b27cd5467d?auth=e9de47e43115f0512a3a19ee48725281">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8a746c87d04c402c8987ef923520c016?auth=2c72370a852708a03e03154b91f766f2">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253827">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:323: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:424: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:139: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253829">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div># STEP: Check that can login with different user</div><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:236: in test_restore_multiaccount_with_waku_backup_remove_switch
    self.sign_in.show_profiles_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `show-profiles` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#349">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253804">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Getting PN by 'user_2' </div><div>Device 2: Looking for a message by text: user_2 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:874: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Can not edit a message with a mention
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea51a8ce7e4d4f9d85f8df9bafd7ec5b?auth=863e4d86d862d14a88f406c61de49778#130">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f15c6199bed346b4a7cdda9b6e2b81ce?auth=97717c14c025d9f91939b439ddc40281#101">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (33)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253818">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1622ba98c0e74fc896d657a307a8e4a5?auth=6f15ce56c644ff0fd283d5c16ff7abc5#92">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9500814a86f146eca5473818d395ddee?auth=3ea446efdef7031617df012f927e45c9#91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253817">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1622ba98c0e74fc896d657a307a8e4a5?auth=6f15ce56c644ff0fd283d5c16ff7abc5#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9500814a86f146eca5473818d395ddee?auth=3ea446efdef7031617df012f927e45c9#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253801">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#131">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253798">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#412">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#344">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253808">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#262">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253800">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#193">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#153">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253819">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#325">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#241">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253834">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#352">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#283">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253795">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/00b2a3ef5d394e7ba92155fef3918b01?auth=d2b1c9a545eacbf98c8b9259ecdf08c3#79">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a459a79397cb43c0ae78782774975c53?auth=9e26cb167b374acc463ff725935417f8#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253830">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#213">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#162">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253820">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#238">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#231">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253811">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#285">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#287">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253828">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#575">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#432">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253822">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#392">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#353">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253810">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#192">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253809">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#150">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253814">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#307">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253813">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253812">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d33b2d2c5eb48ef9f24df05640eac8e?auth=5dde2dff7cf0bd17094733f9695a13eb#352">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1b78f2e770546e6bfc387815afcbe2f?auth=6b893730dce702e09a91c2715ea01257#343">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253803">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/02a8ca533ec944de9b41a6358e00a965?auth=00b7cb20abd86f0aac42010f735a5216#203">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c48d30fc68c242a6bdfc137fe7099eaa?auth=207a296a1e83a8474ab07a1fafd688ba#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253836">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/02a8ca533ec944de9b41a6358e00a965?auth=00b7cb20abd86f0aac42010f735a5216#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c48d30fc68c242a6bdfc137fe7099eaa?auth=207a296a1e83a8474ab07a1fafd688ba#190">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253832">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e686b1ad21d141c68d8c3dcdab1b5ca1?auth=5b1c77575b8c418f46c6d0fa6e479ebb#180">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5e4a58fa01943d99574f80f054382aa?auth=982415fd20a0ad52e2facd892fc9440c#136">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8e3d9a433454cca8eaca66bc9b9b366?auth=23a2012cb941c4066dbde2ec7cfde903#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253805">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e686b1ad21d141c68d8c3dcdab1b5ca1?auth=5b1c77575b8c418f46c6d0fa6e479ebb#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5e4a58fa01943d99574f80f054382aa?auth=982415fd20a0ad52e2facd892fc9440c#73">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8e3d9a433454cca8eaca66bc9b9b366?auth=23a2012cb941c4066dbde2ec7cfde903#73">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253823">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/877a457b3a0a45f9acffa7a98628f7a7?auth=581352697113dbe862a59f29462c0cdb#139">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7a37b0e1a31445b812ff7c39b5f9e29?auth=ce5b2a3f949c6bc587f482ceaa45680d#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253824">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/877a457b3a0a45f9acffa7a98628f7a7?auth=581352697113dbe862a59f29462c0cdb#163">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7a37b0e1a31445b812ff7c39b5f9e29?auth=ce5b2a3f949c6bc587f482ceaa45680d#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253799">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253837">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253821">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#198">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253816">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253833">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86ac29f4b02345af8bcd21094fa54bbc?auth=c798b377cff01d0d4fb0d44344e325a1#219">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253807">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea51a8ce7e4d4f9d85f8df9bafd7ec5b?auth=863e4d86d862d14a88f406c61de49778#204">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f15c6199bed346b4a7cdda9b6e2b81ce?auth=97717c14c025d9f91939b439ddc40281#158">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253825">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea51a8ce7e4d4f9d85f8df9bafd7ec5b?auth=863e4d86d862d14a88f406c61de49778#359">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f15c6199bed346b4a7cdda9b6e2b81ce?auth=97717c14c025d9f91939b439ddc40281#330">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3253815">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea51a8ce7e4d4f9d85f8df9bafd7ec5b?auth=863e4d86d862d14a88f406c61de49778#356">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f15c6199bed346b4a7cdda9b6e2b81ce?auth=97717c14c025d9f91939b439ddc40281#296">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

qoqobolo commented:
Thanks for the fixes @smohamedjavid!
Have a couple of questions about the bottom sheet background

### Q1
This is the background color of the drawers on Android, It's still different from what the desirable color (looks too blue and dark). But could you remind me if this is an issue with blur on Android? I think it has been mentioned before that this cannot be fixed, is this still relevant?

![photo_2023-10-11_13-58-12](https://github.com/status-im/status-mobile/assets/67952253/53d5f31c-d465-42b3-b847-197ce9d8f473)

![photo_2023-10-11_13-58-16](https://github.com/status-im/status-mobile/assets/67952253/fab4ba25-2fd5-4d73-ab42-7a5741d8a5a0)

### Q2

In this PR, the background color of the bottom sheet is not updated on the `Create a profile` page (when you add a photo) both on Android and iOS. Is this expected?
<img width="312" alt="Screenshot 2023-10-11 at 14 10 25" src="https://github.com/status-im/status-mobile/assets/67952253/d614fc94-d073-4b9c-a362-0d396150f43c">

<img width="391" alt="Screenshot 2023-10-11 at 14 13 05" src="https://github.com/status-im/status-mobile/assets/67952253/9584ad9a-daa3-4fdd-bb49-b95b55381e74">

### Q3

The color of the bottom sheet on the profile selection page on Android has some kind of artifact, apparently again due to blur.
It's hard to see in the original screenshot, but I brightened it up in a photo editor to make it clear what I'm talking about. Is it possible to fix this?
![photo_2023-10-11_14-15-25](https://github.com/status-im/status-mobile/assets/67952253/2bb6bd18-e495-4e19-a4bb-2dfd1e1c2feb)

![photo_2023-10-11_14-15-34](https://github.com/status-im/status-mobile/assets/67952253/bec6f1b3-d7d6-4f09-bae8-c8a9f89f8df9)

jo-mut commented:
<img src="https://github.com/status-im/status-mobile/assets/28704507/10ede599-b50c-414d-a29e-4039457ebb11" width="325">

there is this space between the bottom sheet and the selected-item

jo-mut commented:
> > there is this space between the bottom sheet and the selected-item
> 
> @jo-mut hm, I can't reproduce it. What device are you using? Were there any specific steps to reproduce? Does the issue persist after relogin/reopening the app?

the issue seems to be persistent 
I am able to reproduce this on ios simulator

1. Launch app
2. Long press a community on community home screen to launch the bottom sheet
3. Notice the extra bottom margin
4. Also when you long press on a shot message item to launch the bottom sheet
5. Notice the extra bottom margin

smohamedjavid commented:
Hi @qoqobolo - Thanks for testing the PR!

Below I have answered your questions

> Q1
This is the background color of the drawers on Android, It's still different from what the desirable color (looks too blue and dark). But could you remind me if this is an issue with blur on Android? I think it has been mentioned before that this cannot be fixed, is this still relevant?

You are right, this is an issue with blur on Android. I have attached a screenshot from a PR build from 27th September (I happened to have it in my pixel)

| Login  | Onboarding
| --- | --- |
| <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/473e0d09-5511-4ea7-989f-f9010afc6cc6" /> | <img width=250 src="https://github.com/status-im/status-mobile/assets/19339952/84c57a09-5e4d-437d-af01-d248c3b4939d" /> |

> Q2
In this PR, the background color of the bottom sheet is not updated on the Create a profile page (when you add a photo) both on Android and iOS. Is this expected?

Yes, The bottom sheet in the "Create a profile" screen is not updated yet. I can update it in this PR if required.

> Q3
The color of the bottom sheet on the profile selection page on Android has some kind of artifact, apparently again due to blur.
It's hard to see in the original screenshot, but I brightened it up in a photo editor to make it clear what I'm talking about. Is it possible to fix this?

I could able to see the issue. The blur actually creates some issues here. I will try to reproduce and fix it in this PR.

smohamedjavid committed:
[Fix] Selected item bottom padding

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

qoqobolo commented:
Thanks for the answers @smohamedjavid ! 

> Yes, The bottom sheet in the "Create a profile" screen is not updated yet. I can update it in this PR if required

It's up to you. It would be nice to fix it as well, but creating a follow-up issue would be okay too 👍

Figma link for reference https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=2316-259092&mode=design&t=kmll7DJYW7pfFoEv-4

Could you also take a look at the issue reported by @jo-mut?
Maybe it's only reproducible on emulators 🤔

smohamedjavid commented:
Hi @qoqobolo

I have fixed the bottom sheet in the `create profile` screen (Q2). 

Additionally, I have fixed the blur issue in Android (Q3). 

<hr />

> Could you also take a look at the issue reported by @jo-mut?

Sure, I have pushed the fix right now. 

Please retest and let me know if you find any issues. Thanks!

smohamedjavid committed:
[Fix] Issue in blur due to shadow

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

smohamedjavid committed:
[Fix] Bottom sheet type in create profile page

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

qoqobolo commented:
@smohamedjavid thank you!

And another bottom sheet using the wrong theme: in the activity center.
How do you feel about fixing this now as well? :)

| PR  | Figma
| --- | --- |
| <img width="378" alt="Screenshot 2023-10-11 at 17 41 53" src="https://github.com/status-im/status-mobile/assets/67952253/61b81199-c39b-4fa2-9a78-3ca9f52e1d3a"> | <img width="318" alt="Screenshot 2023-10-11 at 17 42 20" src="https://github.com/status-im/status-mobile/assets/67952253/fa1a576a-59da-4483-a49c-877789b3daac"> |

https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=5387-526165&mode=design&t=IbnqWzZ1D0GVyw1S-4

smohamedjavid commented:
> And another bottom sheet using the wrong theme: in the activity center.
How do you feel about fixing this now as well? :)

Is this the final one? 😢 

I will push the fix now 👍

smohamedjavid committed:
[Fix] Bottom sheet type in activity center

Signed-off-by: Mohamed Javid <19339952+smohamedjavid@users.noreply.github.com>

qoqobolo commented:
> Is this the final one? 😢

I hope so 😅 
There are also other drawers with the wrong color, but they are only visible in the dark theme. So I decided to report only this one for now. We will add the rest as follow-ups if we find them later.
Thank you very much for another fix! Don't see any other issues.
I'll quickly recheck the latest build and let you know when it's ready to go!

smohamedjavid commented:
> There are also other drawers with the wrong color, but they are only visible in the dark theme. So I decided to report only this one for now. We will add the rest as follow-ups if we find them later.

If the issue is caused due to this PR, I'm happy to fix it in this PR.

> I'll quickly recheck the latest build and let you know when it's ready to go!

Thanks again @qoqobolo! 😄

qoqobolo commented:
> If the issue is caused due to this PR, I'm happy to fix it in this PR.

Yeah, I know! I'm mostly referring to elements that haven't been updated yet.

So, LGTM, PR can be merged.
Thanks for your work @smohamedjavid!
